timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -326,6 +326,20 @@ export class TimeGraphChart extends TimeGraphChartLayer {
         });
         return element as TimeGraphRowElement;
     }
+    
+    getRowVerticalOffset(row: TimelineChart.TimeGraphRowModel): number {
+        const rowComponent: TimeGraphRow | undefined = this.rowComponents.get(row);
+        return (rowComponent ? rowComponent.position.y : 0)
```

Comment:
Can we handle an undefined `rowComponent` better, since I assume `0` can be also a valid value?
The same applies to the method below. Instead of having these two methods, perhaps have a single getter for the TimeGraphRow object?